app/javascript/packs/application.js

```diff
@@ -7,6 +7,7 @@ import Rails from "@rails/ujs"
 import Turbolinks from "turbolinks"
 import * as ActiveStorage from "@rails/activestorage"
```

Comment:
great job changing all requires to imports.

Reply:
Thanks